product/views/ManageIQ_Providers_CloudManager_AuthKeyPair.yaml

```diff
@@ -21,9 +21,13 @@ db: ManageIQ::Providers::CloudManager::AuthKeyPair
 cols:
 - name
 - fingerprint
+- type
```

Comment:
What is type here?  Was expecting `ext_management_system.name` ?

Reply:
Or nothing at all, like in other cases?

Reply:
type is brought back so we create the correct AuthKey object, so it has the `virtual_belongs_to :ext_management_system` defined